### PR TITLE
feat(Dev/profile): Improved profile page

### PR DESCRIPTION
**Pull request by sid-bagwe**
### Functionality:
Improved profile page

### Solution:
Added two more fields to the Users collection (github and LinkedIn profiles)
Also displayed team name, organization name, manager email and role on the profile page
 

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators: start

### CODE REVIEW

**Review by arjund-06 — APPROVED** [review]
Ship it

File: functions/createNewUserAPI.js [hunk 1]
```diff
@@ -32,10 +32,12 @@ exports.createNewUser = functions.https.onRequest((request, response) => {
                     photoURL: PhotoURL,
                     displayName: DisplayName,
                     email: Email,
-                    phoneNumber: PhoneNumber,
+                    phoneNumber: "",
```
Comment (viv9k) [inline functions/createNewUserAPI.js:35]: Better if we can check if there is a number inside PhoneNumber, is PhoneNumber is null then we must make it empty else we must update it with PhoneNumber

File: src/app/body/profile/profile.component.html [hunk 2]
```diff
@@ -1,15 +1,44 @@
...
+                    <span type="button" class="material-icons float-right" data-toggle="modal" data-target="#exampleModal"
                     data-whatever="@mdo" (click)="editProfile()" style="color: #16C85A;">edit
                     </span>
+                    <h3 class="card-title font-weight-bold font-italic">{{displayName}}</h3>
+                    <h6 class="card-title font-italic">{{role}} at {{organizationName}}, {{teamName}}</h6>
+                    <h6 class="card-subtitle pb-2 border-bottom font-italic text-muted">{{aboutMe}}</h6>
+                    <h6 class="card-title mt-2">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-envelope" viewBox="0 0 16 16">
+                            <path d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v.217l7 4.2 7-4.2V4a1 1 0 0 0-1-1H2zm13 2.383-4.758 2.855L15 11.114v-5.73zm-.034 6.878L9.271 8.82 8 9.583 6.728 8.82l-5.694 3.44A1 1 0 0 0 2 13h12a1 1 0 0 0 .966-.739zM1 11.114l4.758-2.876L1 5.383v5.73z"/>
+                        </svg>
```
Comment (viv9k) [inline src/app/body/profile/profile.component.html:16]: These svg icons could be moved to a common component in icons, just check how it's done. Do this for all..

File: src/app/body/profile/profile.component.html [hunk 3]
```diff
@@ -1,15 +1,44 @@
...
+                        {{email}}
+                    </h6>
+                    <h6 class="card-title">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-telephone" viewBox="0 0 16 16">
+                            <path d="M3.654 1.328a.678.678 0 0 0-1.015-.063L1.605 2.3c-.483.484-.661 1.169-.45 1.77a17.568 17.568 0 0 0 4.168 6.608 17.569 17.569 0 0 0 6.608 4.168c.601.211 1.286.033 1.77-.45l1.034-1.034a.678.678 0 0 0-.063-1.015l-2.307-1.794a.678.678 0 0 0-.58-.122l-2.19.547a1.745 1.745 0 0 1-1.657-.459L5.482 8.062a1.745 1.745 0 0 1-.46-1.657l.548-2.19a.678.678 0 0 0-.122-.58L3.654 1.328zM1.884.511a1.745 1.745 0 0 1 2.612.163L6.29 2.98c.329.423.445.974.315 1.494l-.547 2.19a.678.678 0 0 0 .178.643l2.457 2.457a.678.678 0 0 0 .644.178l2.189-.547a1.745 1.745 0 0 1 1.494.315l2.306 1.794c.829.645.905 1.87.163 2.611l-1.034 1.034c-.74.74-1.846 1.065-2.877.702a18.634 18.634 0 0 1-7.01-4.42 18.634 18.634 0 0 1-4.42-7.009c-.362-1.03-.037-2.137.703-2.877L1.885.511z"/>
+                        </svg>
```
Comment (viv9k) [inline src/app/body/profile/profile.component.html:22]: These svg icons could be moved to a common component in icons, just check how it's done. Do this for all..

File: src/app/body/profile/profile.component.html [hunk 4]
```diff
@@ -1,15 +1,44 @@
...
+                        {{phoneNumber}}
+                    </h6>
+                    <h6 class="card-title">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-building" viewBox="0 0 16 16">
+                            <path fill-rule="evenodd" d="M14.763.075A.5.5 0 0 1 15 .5v15a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5V14h-1v1.5a.5.5 0 0 1-.5.5h-9a.5.5 0 0 1-.5-.5V10a.5.5 0 0 1 .342-.474L6 7.64V4.5a.5.5 0 0 1 .276-.447l8-4a.5.5 0 0 1 .487.022zM6 8.694 1 10.36V15h5V8.694zM7 15h2v-1.5a.5.5 0 0 1 .5-.5h2a.5.5 0 0 1 .5.5V15h2V1.309l-7 3.5V15z"/>
+                            <path d="M2 11h1v1H2v-1zm2 0h1v1H4v-1zm-2 2h1v1H2v-1zm2 0h1v1H4v-1zm4-4h1v1H8V9zm2 0h1v1h-1V9zm-2 2h1v1H8v-1zm2 0h1v1h-1v-1zm2-2h1v1h-1V9zm0 2h1v1h-1v-1zM8 7h1v1H8V7zm2 0h1v1h-1V7zm2 0h1v1h-1V7zM8 5h1v1H8V5zm2 0h1v1h-1V5zm2 0h1v1h-1V5zm0-2h1v1h-1V3z"/>
+                        </svg>
```
Comment (viv9k) [inline src/app/body/profile/profile.component.html:29]: These svg icons could be moved to a common component in icons, just check how it's done. Do this for all..

File: src/app/body/profile/profile.component.html [hunk 5]
```diff
@@ -1,15 +1,44 @@
...
+                        Managed by {{ managerEmail }}
+                    </h6>
+                    <a href="{{githubProfile}}" class="card-link" target="_blank">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-github" viewBox="0 0 16 16">
+                            <path d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.012 8.012 0 0 0 16 8c0-4.42-3.58-8-8-8z"/>
+                        </svg>
```
Comment (viv9k) [inline src/app/body/profile/profile.component.html:35]: These svg icons could be moved to a common component in icons, just check how it's done. Do this for all..

File: src/app/body/profile/profile.component.html [hunk 6]
```diff
@@ -1,15 +1,44 @@
...
+                    </a>
+                    <a href="{{linkedInProfile}}" class="card-link" target="_blank">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-linkedin" viewBox="0 0 16 16">
+                            <path d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"/>
+                        </svg>
```
Comment (viv9k) [inline src/app/body/profile/profile.component.html:40]: These svg icons could be moved to a common component in icons, just check how it's done. Do this for all..

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes.

**Review by viv9k — APPROVED** [review]
ship it

**Pull request merged**